pkg/network/render.go

```diff
@@ -385,6 +388,46 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 	return nil
 }
 
+func validateCidrExpansion(prev, next []operv1.ClusterNetworkEntry) bool {
```

Comment:
Can we rename that to `isClusterNetworkChangeSafe`?

Reply:
yup. done